PluginsManager/PluginInstaller.cs

```diff
@@ -41,23 +41,23 @@ private static async Task<QueryResult> QueryRelease(string name, string url, boo
             if (response.StatusCode == HttpStatusCode.Unauthorized)
             {
                 ConsoleUtil.WriteLine($"[PLUGIN MANAGER] Failed to query {url}! Is the GitHub Personal Access Token set correctly? (Status code: {response.StatusCode})", ConsoleColor.Red);
-                return new();
+                return new QueryResult();
             }
 
             if (!response.IsSuccessStatusCode && response.StatusCode != HttpStatusCode.NotFound)
             {
                 ConsoleUtil.WriteLine($"[PLUGIN MANAGER] Failed to query {url}! (Status code: {response.StatusCode})", ConsoleColor.Red);
-                return new();
+                return new QueryResult();
             }
 
-            var data = JsonSerializer.Deserialize<GitHubRelease>(await response.Content.ReadAsStringAsync());
+            var data = await JsonSerializer.DeserializeAsync(await response.Content.ReadAsStreamAsync(), JsonGenerated.Default.GitHubRelease);
```

Comment:
This can be simplified with an extension method from `System.Net.Http.Json`
```suggestion
            var data = await response.Content.ReadFromJsonAsync(JsonGenerated.Default.GitHubRelease);
```